virttest/nfs.py

```diff
@@ -327,9 +327,13 @@ def setup_remote(self):
 
         self.mount_src = "%s:%s" % (self.nfs_server_ip, self.mount_src)
         logging.debug("Mount %s to %s" % (self.mount_src, self.mount_dir))
-        mount_cmd = ssh_cmd + "'mount -t nfs %s %s -o %s'" % (self.mount_src,
-                                                              self.mount_dir,
-                                                              self.mount_options)
+        if self.mount_options:
+            mount_cmd = ssh_cmd + "'mount -t nfs %s %s -o %s'" % (self.mount_src,
+                                                                  self.mount_dir,
+                                                                  self.mount_options)
+        else:
+            mount_cmd = ssh_cmd + "'mount -t nfs %s %s'" % (self.mount_src,
+                                                            self.mount_dir)
```

Comment:
@svirt, Please fix a logging issue in Line 291, where instead of self.nfs_client_ip,
self.nfs_server_ip is logged,

ssh_cmd = "ssh %s@%s " % (self.ssh_user, self.nfs_client_ip)
        logging.debug("Umount %s from %s" %
(self.mount_dir, self.nfs_server_ip))